pages/common/tpp.md

```diff
@@ -0,0 +1,11 @@
+# tpp
+
+> Command-Line based presentation tool.
+
+- View a presentation:
+
+`tpp {{filename}}`
+
+- Output a presentation:
+
+`tpp -t {{type}} -o {{outputname}} {{filename}}`
```

Comment:
This example is not really clear to me. What is type, and why are there both an {{outputname}} and a {{filename}} parameter? What's the difference between the two?